### ISSUE TITLE
Question on "Background tasks" notification settings

### ISSUE THREAD

**Issue by c0d3z3r0** [Etar-Group/Etar-Calendar#1685]
Etar is causing recurring notifications for background tasks like "scheduling new reminders". I have these notifications set to silent, so they don't bug me.

I just realized that there's a note saying "Notifications required for internal sync. You can safely hide this channel."

There are a few issues / commits on that topic but I couldn't figure out what "hide" really means here. Does it just mean "set it to silent" like I did (1) or can one completely disable this notification type without any side effects (2)?

![signal-2024-06-18-215033_002](https://github.com/Etar-Group/Etar-Calendar/assets/6205797/11091b86-bb64-4291-b636-c52d9d40cf7d)

**Comment by c0d3z3r0** [comment]
Ping :-)

**Comment by jspricke** [comment]
Yes that is meant. Could you send a pull request to improve the documentation?

**Comment by hjek** [comment]
There are a lot of those notifications:

![421 notifications per day](https://github.com/user-attachments/assets/c4212a3f-bee1-457d-98b6-7cd290c9e3a6)
